### PR TITLE
Refactor show_sort_info function

### PR DESCRIPTION
**Pull request by TakteS**

### CODE REVIEW

File: web/templates/package/index.html.eex [hunk 1]
```diff
@@ -59,6 +59,9 @@
         <li role="presentation">
           <a role="menuitem" tabindex="-1" href="<%= package_path(HexWeb.Endpoint, :index, sort_created_params) %>">Recently created</a>
         </li>
+        <li role="presentation">
+          <a role="menuitem" tabindex="-1" href="<%= package_path(HexWeb.Endpoint, :index, sort_updated_params) %>">Recently updated</a>
+        </li>
```
Comment (ericmj) [inline web/templates/package/index.html.eex]: Nice. Seems like we do dropped this at some point when rebasing the frontend branch.

File: web/templates/package/index.html.eex [hunk 2]
```diff
@@ -8,7 +8,7 @@
     <ul>
       <%= for letter <- @letters do %>
       <li>
-        <a href="<%= package_path(HexWeb.Endpoint, :index, letter: letter) %>">
+        <a href="<%= package_path(HexWeb.Endpoint, :index, letter: letter, sort: @sort) %>">
```
Comment (ericmj) [inline web/templates/package/index.html.eex]: Does it make sense to support sorting on anything different than name when using letters? Maybe the selector should be removed?

File: web/views/package_view.ex [hunk 3]
```diff
@@ -2,11 +2,9 @@ defmodule HexWeb.PackageView do
...
-      "name" => "name", "inserted_at" => "recently created",
-      "downloads" => "downloads", "updated_at" => "recently updated"}
-
-    if Map.has_key?(available_params, param), do: "(Sorted by #{available_params[param]})"
-  end
+  def show_sort_info("name"), do: "(Sorted by name)"
+  def show_sort_info("inserted_at"), do: "(Sorted by recently created)"
+  def show_sort_info("updated_at"), do: "(Sorted by recently updated)"
+  def show_sort_info("downloads"), do: "(Sorted by downloads)"
+  def show_sort_info(param), do: nil
```
Comment (ericmj) [inline web/views/package_view.ex]: How come we don't have to handle `nil` any more?

Comment (TakteS) [inline web/views/package_view.ex]: We need to do this: "def show_sort_info(param), do: nil" for cases when user try sorting by the unavilable param. Without this we have an error.

And "def show_sort_info(nil), do: "(Sorted by name)"" for cases when the sorting aren't selected

**Comment by ericmj** [comment]
:heart:

**Pull request merged**